### DISCUSSION TITLE
Capture traffic with Alfa Network AWUS036NHA

### DISCUSSION THREAD

**Discussion by gbernaldo82** [aircrack-ng/aircrack-ng#2612] [Q&A] [answered]
Hi, good morning!

This is my first question, so, please, if this is not the correct place to ask this question, just let me know about it.

I have a device Alfa Network AWUS036NHA configured on Kali Linux as Monitor Mode. This works just fine, I mean, after launching 
`sudo airmon-ng start wlan0`

I see the device is configured correctly:

```
┌──(XXXXX㉿kali)-[~]
└─$ sudo airmon-ng start wlan0   

Found 2 processes that could cause trouble.
Kill them using 'airmon-ng check kill' before putting
the card in monitor mode, they will interfere by changing channels
and sometimes putting the interface back in managed mode

    PID Name
    651 NetworkManager
    855 wpa_supplicant

PHY     Interface       Driver          Chipset

phy0    wlan0           ath9k_htc       Qualcomm Atheros Communications AR9271 802.11n
                (mac80211 monitor mode vif enabled for [phy0]wlan0 on [phy0]wlan0mon)
                (mac80211 station mode vif disabled for [phy0]wlan0)


```
So it seems that everithing is correct. 

Now the problem I have is the next. I have one mobile phone which has installed one application from our client and sometimes that application should send commands to our server but I am not able to see such traffic in our system. That application should send us REST Api commands to our server but I don't see that traffic. So, I launched a wireshark instance in the interface (SERVER SIDE) where such traffich should occur but there is no traffic. Our client says the problem is with some kind of bridge/hub/router/whatever doing some kind of filtering in our lab. I don't think that's the problem cause we've tried disabling the firewall, that's why I need to sniff traffic from the mobile phone. 

Now, after running the airmon-ng start wlan0 and after starting wireshark over the device wlan0mon I just see packets with the protocol 802.11. I don't see anything more. I thought using the device in promisucuous mode I will have access to everypacket on the wifi network (like in a ethernet device in monitor mode). 

So my question is about how can I get all traffic packages. I've followed some guides about getting started but most guides are about packet injection and cracking wifi networks which are not the case in this scenario cause all the devices are on the same network.

Can anyone help me with this problem please?

Thank you so much in advance!

**Comment by Mister-X-** [comment]
That adapter can capture only 1 stream in 802.11n. So you'd have to ensure your router is 1. set-up with 802.11n only (not ac or ax), 2. It has only one stream, and 3. That it's set on a 2.4GHz channel.

Second thing: Being too far or too close from either device will result in lost packets. Be at least a meter (or yard) or 2 away from both devices.

Third thing: kill the network managers before putting the adapter in monitor mode, it will start changing channels.

**Reply by gbernaldo82** [reply]
Thank you so much for your answer, it's really appreciated!

Let me check out that parameters on the router with the sysadmin.

I'll tell you something as soon I've done all your steps.

**Reply by gbernaldo82** [reply]
Hi again @Mister-X- 

I've been talking with our sysadmin about the wifi network configuration and he said to me that the configuration is:

802.11n in 2,4GHz and 802.11ac in 5GHz.

In regards to second thing, both devices are near one from the other (distance is <1m) 
In regards to third thing, I killed the network manager with the command:

`airmon-ng check kill`

Then I setup the wifi in monitor mode:

`airmon-ng start wlan0`

After that I launched the wireshark and started the capture but I only get packets with protocol 802.11. I mean, the same thing as before.

Any idea?

Thank you so much!

**Reply by Mister-X-** [reply, marked as answer]
Yes. I'm repeating myself:

- This wireles card is 2.4GHz only, so if the device connects on 5ghz, you'll get nothing
- This wireless card is 802.11n only, so if the device connects in 802.11ac, you'll get nothing
- This wireless card is 1 stream, so if the device negotiated with the AP a 2-stream, you'll get nothing
- If you're too close to the device, signal will be too strong, and you'll get nothing.

In any case, that doesn't make sense to capture on the wireless side, just put a tap in the network between the AP and the end point. Or do the capture on the server itself.

**Reply by gbernaldo82** [reply]
Ok, thank you so much @Mister-X- !

Sorry for any inconvenience

**Reply by Mister-X-** [reply]
There is no inconvenience. It's just that I initially mentioned why you possibly didn't get anything, and you seem to have still done it afterward, and that is disappointing.
